sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio_test.py

```diff
@@ -338,6 +346,26 @@ def test_DatastoreWriteLargeEntities(self):
 
       self.assertEqual(2, commit_count[0])
 
+  def test_DatastoreWrite_monitoring_info(self):
+    num_entities_to_write = 1
+    self.check_DatastoreWriteFn(num_entities_to_write)
+    resource = resource_identifiers.DatastoreNamespace(self._PROJECT, None)
+    labels = {
+        monitoring_infos.SERVICE_LABEL: 'Datastore',
+        monitoring_infos.METHOD_LABEL: 'BatchDatastoreWrite',
+        monitoring_infos.RESOURCE_LABEL: resource,
+        monitoring_infos.DATASTORE_NAMESPACE_LABEL: None,
+        monitoring_infos.DATASTORE_PROJECT_ID_LABEL: self._PROJECT,
+        monitoring_infos.STATUS_LABEL: 'ok'
+    }
+
+    metric_name = MetricName(
+        None, None, urn=monitoring_infos.API_REQUEST_COUNT_URN, labels=labels)
+    metric_value = MetricsEnvironment.process_wide_container().get_counter(
```

Comment:
Would you mind testing it like this:
https://github.com/apache/beam/blob/39cf3fccbb3950b9fd0c8938301d192c3ca85cce/sdks/python/apache_beam/io/gcp/bigquery_tools_test.py#L436

Specifically, pulling out the MonitoringInfos objects with the call to to_runner_api_monitoring_infos and testing that instead of testing with get_counter.

The point if this is to make sure that the monitoring_infos generated and send to the worker_harness are as we exepct

Reply:
Thanks for your comments. I changed the test to match the example assertions https://github.com/apache/beam/blob/1e994538fb848f7175f810d5d2f69ea36bc00d66/sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio_test.py#L183